### PR TITLE
Update event endpoint response `end` -> `end_at`

### PR DESCRIPTION
**Pull request by wubini**
https://www.pivotaltracker.com/story/show/168674434

### CODE REVIEW

File: source/includes/_authenticated_api_events.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Events represent real-world or virtual meetings that members can RSVP to. RSVP r
     "url": "https://demo.controlshiftlabs.com/events/deliver-the-petition-to-the-wizard",
     "description": "Join us as we travel to the Emerald City to deliver our petition to the Wizard.",
     "start": "2018-03-01T12:00:00Z",
```
Comment (lavaturtle) [inline source/includes/_authenticated_api_events.md]: should this also get changed to `start_at`?

Comment (wubini) [reply source/includes/_authenticated_api_events.md]: yes!

Comment (lavaturtle) [reply source/includes/_authenticated_api_events.md]: (there's also a handful of `start`s in `json_local`)

**Review by lavaturtle — APPROVED** [review]
:+1:

**Pull request merged**